src/react_native/status.cljs

```diff
@@ -0,0 +1,47 @@
+(ns react-native.status
+  (:require ["react-native" :as react-native]
```

Comment:
and also please move all methods which are also used in status-im namespace

Reply:
will do

Reply:
reverted related change, created a separate issue https://github.com/status-im/status-mobile/issues/14581